### PR TITLE
feat: Add contact session feature

### PR DESCRIPTION
**Pull request by Reconfort**
## What does this PR do?
1. Creating Contact section

## How can this changes be manually tested
1. Clone this project
3. Navigate into cloned Project
4. run `npm install`
5. open index.html in browser

### CODE REVIEW

**Review by frempongdev — COMMENTED** [review]
Hi @Reconfort, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❗ ❗ 

You have submitted a project, with a broken link in the README file. Your README also does not have a LICENSE section. 🔢 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -234,7 +234,20 @@ <h2>Skills</h2>
   </ul>
   </div>
 </section>
-    </div>
+
+<section class="contact">
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+    I’d love to help with it!</p>
+  <form class="contact-form" action="https://formspree.io/danny.reco2002@gmail.com" method="POST">
```
Comment (IngDavidTM) [inline index.html:242]: - [ ] While you did a fantastic job the form is not working as expected as you can see below. The problem is that the `action` value should be similar to `https://formspree.io/f/{form_id}`. [Here](https://formspree.io/html/) you can find more information about how to set it up on your project :+1:.

Screenshot from 2023-07-18 18-37-20

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Reconfort 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Reconfort,

✔️ Implementation of submission with external service

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**